packages/react-bindings/src/hooks/useStateManager.ts

```diff
@@ -12,7 +12,7 @@ const getDefinedProps = <Props extends Record<string, any>>(props: Props): Parti
 
   Object.keys(props).forEach(propName => {
     if (props[propName] !== undefined) {
-      definedProps[propName] = props[propName]
+      ;(<Record<string, any>>definedProps)[propName] = props[propName]
```

Comment:
Was this applied when you saved in your editor? Seems inconsistently enforced to only changed lines.

Reply:
That's because of the added `(cast)`.
https://prettier.io/docs/en/rationale.html#semicolons